graphql/queryResolvers/getLessonMentors.ts

```diff
@@ -0,0 +1,24 @@
+import { User, UserLesson } from '../../helpers/dbload'
+
+type ArgsGetLessonMentors = {
+  lessonId: string
+}
+
+export const getLessonMentors = async (
+  _parent: void,
+  args: ArgsGetLessonMentors
+) => {
+  const { lessonId } = args
+  try {
+    const results = await UserLesson.findAll({
+      where: { lessonId },
+      include: [{ model: User }]
+    })
+
+    return results.map((result: any) => {
+      return result.User
```

Comment:
does this contribute to a leak of personal info? Like... could I specify `email` and then get user's email?

Reply:
https://github.com/garageScript/c0d3-app/blob/master/graphql/typeDefs.ts#L115 It does. Now anyone can get anyone's email for spam, not good I think.

Reply:
```
return {
  username: result.User.username
  ...
}
```

Reply:
Hmm, yes fair point. I'll return just the username.